src/core/field/Opcode.h

```diff
@@ -1530,8 +1530,9 @@ STRUCTPACK(struct OpcodeLABEL : public OpcodeBase {
 	quint16 _label;
 });
 
-class Opcode
+class Opcode : public QObject
 {
+	Q_OBJECT
```

Comment:
See the main PR description about this part

Reply:
I don't want to use this macro, as it adds too much overhead, I only need a tr() function

Reply:
lupate is the only one to complain in its code analysis

Reply:
Ok shall I revert this commit then? The main issue is that the compiler says the class needs `Q_OBJECT` defined. The whole warning starts from this line of code: https://github.com/julianxhokaxhiu/makoureactor/blob/master/src/core/field/Opcode.cpp#L2093

Reply:
Ok dropped the patch entirely, thank you!

Reply:
About this file 

1. Do Not include a module like is done in this class with `#include <QtCore>` . Instead only include the needed objects in this case  QObject and QString . This will reduce compile time The entire project should be checked for this . 
2. Never use the Q_OBJECT macro in a class that is not subclased from QObject. If you need to you can use `Q_GADGET` but for this class just call `QObject::tr`. and then there is not a need for moc so no reason to have it run on the object.